<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaClipSelector</class>
 <widget class="QMainWindow" name="MediaClipSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>739</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>683</width>
    <height>686</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Select media clip</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="media_path_combobox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QTimeEdit" name="start_timeedit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QTimeEdit" name="end_timeedit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="set_start_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Set current position as start point</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="load_disc_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Load disc</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="play_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../images/media_playback_start.png</normaloff>../images/media_playback_start.png</iconset>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="end_point_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>End point</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QComboBox" name="subtitle_tracks_combobox">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="title_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QComboBox" name="audio_tracks_combobox">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="set_end_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Set current position as end point</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="save_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Save current clip</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QPushButton" name="close_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="start_point_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Start point</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="jump_start_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Jump to start point</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="audio_track_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Audio track</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QTimeEdit" name="media_position_timeedit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="QFrame" name="media_view_frame">
      <property name="minimumSize">
       <size>
        <width>665</width>
        <height>375</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="subtitle_track_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Subtitle track</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="jump_end_pushbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Jump to end point</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="media_path_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Media path</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QComboBox" name="title_combo_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentText" stdset="0">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="3">
     <widget class="QSlider" name="position_horizontalslider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>media_path_combobox</tabstop>
  <tabstop>load_disc_pushbutton</tabstop>
  <tabstop>title_combo_box</tabstop>
  <tabstop>audio_tracks_combobox</tabstop>
  <tabstop>subtitle_tracks_combobox</tabstop>
  <tabstop>play_pushbutton</tabstop>
  <tabstop>position_horizontalslider</tabstop>
  <tabstop>media_position_timeedit</tabstop>
  <tabstop>start_timeedit</tabstop>
  <tabstop>set_start_pushbutton</tabstop>
  <tabstop>jump_start_pushbutton</tabstop>
  <tabstop>end_timeedit</tabstop>
  <tabstop>set_end_pushbutton</tabstop>
  <tabstop>jump_end_pushbutton</tabstop>
  <tabstop>save_pushbutton</tabstop>
  <tabstop>close_pushbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
